.buildkite/release-automation/verify-macos-wheels.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -euo pipefail
+
+set -x
+
+python_versions=("3.8" "3.9" "3.10" "3.11")
```

Comment:
nit: caps for bash vars: `PYTHON_VERSIONS`

Reply:
fixed!